packages/app-check/src/client.ts

```diff
@@ -42,17 +42,18 @@ interface AppCheckRequest {
 
 export async function exchangeToken(
   { url, body }: AppCheckRequest,
-  platformLoggerProvider: Provider<'platform-logger'>
+  heartbeatServiceProvider: Provider<'heartbeat'>
 ): Promise<AppCheckTokenInternal> {
   const headers: HeadersInit = {
     'Content-Type': 'application/json'
   };
   // If platform logger exists, add the platform info string to the header.
-  const platformLogger = platformLoggerProvider.getImmediate({
+  const heartbeatService = heartbeatServiceProvider.getImmediate({
```

Comment:
probably should update the comment above since it reference platform logger which was renamed